README.rst

```diff
@@ -1,207 +1,192 @@
+pygame
```

Comment:
What do you think about removing this header? It takes up quite a lot of space, and most people will know this is about pygame.

Reply:
Personally I like the way it looks, especially to compliment the badges to be displayed right below.  Not a strong opinion though, just that I'm used to see many READMEs looking that way.